domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/complexfield/ComplexFieldEntityShowConditionValidatorImpl.java

```diff
@@ -37,7 +38,7 @@ public ValidationResult validate(ComplexFieldEntity complexField, ValidationCont
         }
 
         showCondition.getFields().forEach(showConditionField -> {
-            if (!complexField.getComplexFieldType().hasComplexField(showConditionField)) {
+            if (!complexField.getComplexFieldType().hasComplexField(showConditionField) && !MetadataField.isMetadataField(showConditionField)) {
```

Comment:
An alternative we could consider also would be to add a method to ShowCondition called `getNonMetadataFields()` or `getCaseFields()` and change the existing code with: 
```showCondition.getNonMetadataFields().forEach```

Reply:
same suggestion for the other places